### DISCUSSION TITLE
RTSP

### DISCUSSION THREAD

**Discussion by dionchua** [ant-media/Ant-Media-Server#5813] [Q&A]
Hi all, I want to be able integrate RTSP streaming on an application and have recently tried to create a new stream source broadcasting with the stream URL pointing to the RTSP link but it seems like it is always offline.
Not sure if I am heading towards the right direction and will like to check what are the possible ways for me to view RTSP through the server?

**Comment by yashtandon113** [comment]
@dionchua 
By default, the RTSP output is not available on the server. You can only ingest the RTSP URL as a stream source.

In order to get the RTSP output from the server, you can use the **GST-Ant Fusion** plugin on [Ant Media Marketplace](https://antmedia.io/marketplace/). You can request a demo call if you are interested.

Please checkout the related blogpost post **[here](https://antmedia.io/how-to-live-stream-rtsp-output-with-ant-media-server/)**.

I hope it helps.
